packages/cardano-services-client/src/TxSubmitProvider/txSubmitHttpProvider.ts

```diff
@@ -47,7 +47,7 @@ export const txSubmitHttpProvider = (config: CreateHttpProviderConfig<TxSubmitPr
           return mapHealthCheckError(error);
         }
         case 'submitTx': {
-          if (typeof error === 'object' && typeof error.innerError === 'object') {
+          if (error && typeof error === 'object' && typeof error.innerError === 'object') {
```

Comment:
It's a good practice to add or update a test when making a fix

Reply:
Removed from this PR, Should be addressed with ADP-2397